draft-ietf-ppm-dap.md

```diff
@@ -639,9 +639,12 @@ struct {
 struct {} Empty;
 ~~~
 
-DAP uses the 16-byte `ReportID` as the nonce parameter for the VDAF
-`shard` and `prep_init` methods (see {{!VDAF, Section 5}}). Thus for a VDAF to
-be compatible with DAP, it MUST specify a `NONCE_SIZE` of 16 bytes.
+DAP uses the 16-byte `ReportID` as the nonce parameter for the VDAF `shard` and
+`prep_init` methods (see {{!VDAF, Section 5}}). Additionally, DAP includes
+messages defined in the VDAF specification encoded as opaque byte strings within
+various DAP messages. Thus, for a VDAF to be compatible with DAP, it MUST
+specify a `NONCE_SIZE` of 16 bytes, and must specify encodings all relevant
```

Comment:
"…must specify encodings all relevant…" probably is missing a word, but I'm not sure what word was intended